README.md

```diff
@@ -135,8 +136,17 @@ gcloud iam service-accounts keys create key.json \
   --iam-account $GCPSA_NAME@$GCP_ORG_NAME.iam.gserviceaccount.com
 
 
-# billindacount.txt contains billing ID from https://console.cloud.google.com/billing
-kubectl create secret generic gcp-project-operator --from-file=key.json=secrets/key.json --from-file=billingaccount=secrets/billingaccount.txt -n gcp-project-operator
+kubectl create secret generic gcp-project-operator --from-file=key.json=secrets/key.json -n gcp-project-operator
+
+```
+
+## GCP Operator configmap creation
+
+```bash
+export ORGPARENTFOLDERID="240634451310" # Google Cloud organization Parent Folder ID
```

Comment:
```suggestion
export ORGPARENTFOLDERID="292670943244" # OpenShift Dedicated DEV folder
```

Reply:
I'd prefer to use a fictive folder ID

Reply:
Sure, we can remove our actual IDs and have a separate private gdoc on how to set configure the operator to run with our gcp env.

Reply:
I can remove the actual ID here but adding private gdoc may be handled in another task.

Reply:
I'll do the private gdoc.